toss_4_x86_64_ib_cray/compilers.yaml

```diff
@@ -116,3 +116,16 @@ compilers::
     modules: []
     environment: {}
     extra_rpaths: []
+- compiler:
+    spec: clang@15.0.0
```

Comment:
Should these be called `amdclang` now?

Reply:
I don't think `amdclang` is a recognized spack compiler name. Though searching through their repo I did find this:

https://github.com/spack/spack-stack-catalog/blob/fb9aca476c429e8574853eee98eaf67c6753c5d8/_stacks/eugeneswalker/facility-spack/crusher/22.11/PrgEnv-amd/rocm-rocmcc/spack.yaml#L11-L16

Reply:
Oh jeez, too many compiler names...

Reply:
@becker33 - should we use `rocmcc` as the name for this compiler?

Reply:
Should I rename to `rocmcc` then?

Reply:
Also, referring to my question in the description, this may only work for rocm@5.4.1, right? If I wanted to use rocm@5.4.3, I may need to use that rocm's amdclang compiler instead even though it claims to also be called "Clang v15.0.0".

Reply:
We should also have a look to the way Rob Blake configured rocm stack in his confluence page. (I’d have to retrieve the link...).

Reply:
I just got confirmation from Greg Becker that we should be using `rocmcc@5.4.3` for the amdclang at `/opt/rocm-5.4.3/llvm/bin/amdclang` and likewise for `rocmcc@5.4.1` for the amdclang at `/opt/rocm-5.4.1/llvm/bin/amdclang`

Reply:
I'll make those changes @davidbeckingsale @white238 @adrienbernede

Reply:
Thanks Chris for pointing that out!